recipe/meta.yaml

```diff
@@ -44,6 +44,8 @@ test:
   # https://github.com/conda-forge/staged-recipes/pull/292#issuecomment-208080893.
   commands:                # [not win]
     - nosetests zmq.tests  # [not win]
+  imports:                 # [win]
+    - zmq                  # [win]
```

Comment:
Well, it looks like the same `zeromq` version and build number in the last successful build here and the failure in `ipyleaflet`. We did update the `vs2015_runtime` version yesterday. Maybe that is to blame?